movieposts/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class MoviePost(models.Model):
+    user       = models.ForeignKey('users.User', on_delete = models.CASCADE)
+    movie      = models.ForeignKey('movies.Movie', on_delete = models.CASCADE)
+    content    = models.TextField()
+    like       = models.IntegerField()
+    created_at = models.DateTimeField(auto_now_add = True)
+    updated_at = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        db_table = "movieposts"
```

Comment:
영화에 대한 리뷰를 관리하는 테이블이라면 이름을 `Review`로 더욱 명확하게 만들어주시는게 좋을 것 같습니다!
어떠신가요?

```suggestion
class Review(models.Model):
    user       = models.ForeignKey('users.User', on_delete = models.CASCADE)
    movie      = models.ForeignKey('movies.Movie', on_delete = models.CASCADE)
    content    = models.TextField()
    like       = models.IntegerField()
    created_at = models.DateTimeField(auto_now_add = True)
    updated_at = models.DateTimeField(auto_now = True)

    class Meta:
        db_table = "movieposts"
```

Reply:
메가박스 사이트에서 해당 메뉴가 무비포스트라고 되어있어서 직관적으로 표현하기 위해 이렇게 했습니다. 혹시 바꾸는게 나을까요?
![스크린샷 2021-09-15 오전 10 38 29](https://user-images.githubusercontent.com/42742076/133356265-8a02c561-4686-4d4d-8834-bfeda74df21e.png)